kettle/README.md

```diff
@@ -70,6 +70,22 @@ It might take a couple of hours to be fully functional and start updating BigQue
 
 To add fields to the BQ table, Visit the [k8s-gubernator:build BigQuery dataset](https://bigquery.cloud.google.com/dataset/k8s-gubernator:build) and Select the table (Ex. Build > All). Schema -> Edit Schema -> Add field. As well as update [schema.json](./schema.json)
 
+## Adding Buckets
+
+To add a new GCS bucket to Kettle, simply update [buckets.yaml](./buckets.yaml) in `master`, it will be auto pulled by Kettle on the next cycle.
+
+```yaml
+gs://<bucket path>: #bucket url
+  contact: "username" #Git Hub Username
+  prefix: "abc:" #the identifier prefixed to jobs from this bucket (ends in :).
+  sequential: (bool) #an optional boolean that indicates whether test runs in this
+  #                  bucket are numbered sequentially
+  exclude_jobs: [
+    'job_name1',
+    'job_name2',
+  ] #List of jobs to explicitly exclude from kettle data collection
```

Comment:
For a multi-line list I would probably use a different syntax unless there's a good reason not too
```suggestion
  exclude_jobs: # list of jobs to explicitly exclude from kettle data collection
    - job_name1
    - job_name2
```

Reply:
ah thanks, I just went with the first option I saw when searching `adding a list to yaml`